### ISSUE TITLE
Link format makes it tough to tell what's a link or not 

### ISSUE THREAD

**Issue by srsexton94** [freeCodeCamp/LearnToCodeRPG#32]
### What bug did you encounter?

On the August 3 call to the Annika character there's what looks like a link to https://developerquiz.org, but it doesn't seem to link anywhere

In general accessibility-wise I know the advice is to not distinguish interactive elements solely by one characteristic (ie color) and this is all the more important for links in bodies of text (ie some color-blind users won't be able to tell these are meant to be links at all.) 

Currently, I believe your links fail WCAG 1.4.1 - I'd recommend adding an underline to links and only applying that styling to things that are actually linking somewhere.  Ideally, you might also consider making the color more distinct from your body text and making the cursor a pointer when over links (not sure if that's an accessibility issue but I know it confused me haha)

### Steps to Reproduce

Play the game to august 3rd, get on the phone call to Annika, and proceed to the step where she's discussing developer quizzes. Try and click on the link. The game will proceed to the next dialogue slide instead.

**Comment by jdwilkin4** [comment]
Hi @srsexton94 ! 

It is my understanding that the reference to the developer quiz site is not supposed to be an active link.
I think it is just there so people can explore the site later on their own outside of the game. 
But it does make sense to remove the link styling so people aren't tempted to click on it.

**Comment by RuolinZheng08** [comment]
Fixed in a45212e
